hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -216,17 +216,54 @@ static int start_model(OtbnModel &model, unsigned start_addr) {
   return 0;
 }
 
+// Extract 256-bit RND EDN data from a 4 state logic value. RND data is placed
+// into 8 element uint32_t array dst.
+static void set_rnd_data(uint32_t dst[8], svLogicVecVal *src) {
```

Comment:
Can you make the second argument a `const svLogicVecVal *`? (Or even a const array of 8 elements, to make it clear how much we're reading?)